### DISCUSSION TITLE
About Unicode : How to understand Korean Language with Euphony?

### DISCUSSION THREAD

**Discussion by designe** [euphony-io/euphony#44] [Knowledge Share]
As you know that, it doesn't support Korean Language.
So, let me share some Unicode development information.

### About ASCII
All of you are familiar with ASCII codes. When you learn C language in your freshman year of college, you must have experienced that each letter is matched with a number. 'a' is stored as 61, 'b' as 62, and 'c' as 63.
   ASCII code is stored as a total of 128 characters from 0 to 127. With only 7 bits, everything can be expressed. This means that it is stored in one byte. Everyone knows that a char type is 1 byte. ASCII is expressed with only 7 bits, so it is neatly stored in a char. All you need to know about ASCII code is this.

### References
https://stackoverflow.com/questions/50403342/how-do-i-properly-use-stdstring-on-utf-8-in-c
https://modoocode.com/292
https://github.com/euphony-io/euphony/discussions/43#discussioncomment-1321791

If you interested in unicode, Please contribute by patching the code! 

- Related Tag : cpp, std::string, charset, ASCII, UTF-8, UTF-16, UTF-32, Unicode

### Description
 - Currently, Euphony library supports 2 types charsets such as DefaultCharset, ASCIICharset.
 - In case of the DefaultCharset, Hex Value Itself!
 - In case of the ASCIICharset,  English & some symbols can be expressed.
 - In euphony, Follow the Charset interface!
 - We can make UTF8, UTF16, UTF32, and so on.

**Comment by judemin** [comment]
Thx for the discussion!
